Unicode/Makefile.am

```diff
@@ -67,6 +67,15 @@ libgunicode_la_SOURCES =	\
 	gwwiconv.c		\
 	$(NULL)
 
+libgunicode_la.$(OBJEXT):	\
```

Comment:
Hi @JoesCat, $(OBJEXT) is unnecessary at best and possibly harmful for future maintainability. I wrote why as a comment to a previous commit, but maybe it got lost. Here my original comment (from <https://github.com/fontforge/fontforge/commit/fe13eca58f262fd4e0d4c84070bddd086ee0399e#commitcomment-19995303>)

> hi @JoesCat, I think I am not getting what you are saying.
>
> Just having a `.h` file in `_SOURCES` will never lead to the creation of a separate object file. (And even if it did it would be completely empty, because (at the end of the PR) all the variables in it are declared static.)
>
> You can see that the only rule that depends on `is_Ligature_data.h` or that makes use of it is in `.deps/libgunicode_la-is_Ligature.Plo`:
>
> ```Makefile
> libgunicode_la-is_Ligature.lo: is_Ligature.c is_Ligature_data.h \
>  ../inc/basics.h ../inc/fontforge-config.h ../lib/stdio.h \
>  /usr/include/stdio.h /usr/include/features.h \
> ```
>
> but that is just a declaration of dependency. Only one `.lo` file will be produced. And it will be produced using this ad-hoc rule generated by the autotools (split in multiple lines for readability):
>
> ```Makefile
> libgunicode_la-is_Ligature.lo: is_Ligature.c
> 	$(AM_V_CC)$(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile \
> 	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(libgunicode_la_CPPFLAGS) $(CPPFLAGS) \
> 	$(libgunicode_la_CFLAGS) $(CFLAGS) -MT libgunicode_la-is_Ligature.lo -MD -MP -MF \
> 	$(DEPDIR)/libgunicode_la-is_Ligature.Tpo -c -o libgunicode_la-is_Ligature.lo \
> 	`test -f 'is_Ligature.c' || echo '$(srcdir)/'`is_Ligature.c
> ```
>
> After running `make` you can see that in `.libs/` only one `.o` file for `is_Ligature` has been produced and no traces of `is_Ligature_data.h`. Autotools and libtool may be convoluted and do weird things, but not that weird. ;)

In addition to that comment, I should also point out that all the `../inc` files (and many others) are automatically added as dependencies by Automake. You can see this inspecting the deps file `Unicode/.deps/libgunicode_la-is_Ligature.Plo`.

Please just add `is_Ligature_data.h` to `_SOURCES`. Let's make the autotools do some work for us. ;)